plugin/lspconfig.lua

```diff
@@ -6,17 +6,6 @@ vim.g.lspconfig = 1
 local api, lsp = vim.api, vim.lsp
 local util = require('lspconfig.util')
 
-if vim.fn.has 'nvim-0.8' ~= 1 then
-  local version_info = vim.version()
-  local warning_str = string.format(
-    '[lspconfig] requires neovim 0.8 or later. Detected neovim version: 0.%s.%s',
```

Comment:
this should be bumped to 0.9, not removed entirely

we need an "active" check like this somewhere, to avoid confusion and bug reports.

Reply:
Does the version check in lsoconfig.lua not suffice? I assumed we only needed one check.